pombola/south_africa/views.py

```diff
@@ -2115,7 +2125,7 @@ def download_attendance_data(self):
         if results is None:
             results = []
             while next_url:
-                resp = requests.get(next_url)
+                resp = requests.get(next_url, timeout=API_REQUESTS_TIMEOUT)
```

Comment:
There wasn't one previously for other types of failure, so I was going to leave the behaviour here the same.